src/precomp.hpp

```diff
@@ -168,7 +168,7 @@ namespace cv
  * @note Backends are available only if they have been built with your OpenCV binaries.
  * See @ref videoio_overview for more information.
  * */
-//#ifdef DELETETHIS	
+#if	0
```

Comment:
We should delete this

Reply:
pretty much cut all the unneeded code from that precomp.hpp.  Most of that was artifacts from when I created the videocapture modification to take care of some camera issues including laggy frames.